pages/wiki/package-installation.md

```diff
@@ -7,13 +7,25 @@ layout: documentation
   <h1 id="watchfaceinstallation">Watchface Installation</h1>
 </div>
 <div>
-  <p>Some watchfaces (e.g. <a href="https://github.com/AsteroidOS/unofficial-watchfaces">the unofficial-watchface
-      collection</a>) bring their own installation script.
-    If you wish to install a watchface that has no installation script or is not provided as a package you can install
+  <h4>Scripted method</h4>
+  <p>Watchfaces listed in the <a href="https://github.com/AsteroidOS/unofficial-watchfaces">unofficial-watchface
+      collection</a> bring their own installation script.</p>
+  <div>
+    <p>To use the script you will need to clone the unofficial-watchfaces repo.
+      <pre><code>git clone https://github.com/AsteroidOS/unofficial-watchfaces</code></pre>
+      Change into the new directory.
+      <pre><code>cd unofficial-watchfaces</code></pre>
+      Execute the script with no flag to use SSH connection and SCP commands.
+      <pre><code>./deploy.sh</code></pre>
+      Or use ADB connection and commands with the -a flag.
+      <pre><code>./deploy.sh -a</code></pre></p>
+  </div>
+  <h4>Manual method</h4>   
```

Comment:
I would add a line break above the `<h4>` element for the looks.

Currently:
![without_linebreak](https://user-images.githubusercontent.com/14153182/121749127-6fc19080-cb0a-11eb-8bc2-d892441d2dbf.png)

With `<br/>`:
![with_linebreak](https://user-images.githubusercontent.com/14153182/121749123-6f28fa00-cb0a-11eb-8fc5-bc3e5c918869.png)

Reply:
Absolutely. 
Tbh i did this in blind flight and just copied the code from the block below.
How can i have an environment to see what i even do in Less? Any hint what to setup/install?

Reply:
No worries. Catching things like this is the purpose of reviews!
I just did `hub pr checkout 113 && npx grunt && chromium _asteroidos.org/wiki/package-installation/index.html` to review the pr.